floss/language/go/extract.py

```diff
@@ -47,6 +50,9 @@ def extract_go_strings(
         """
         alignment = 0x8
         fmt = "<II"
+        combinedregex = re.compile(
+            b"\x81\xf9(....)|\x81\x38(....)|\x81\x7d\x00(....)|\x81\x3B(....)|\x66\x81\xf9(..)|\x66\x81\x7b\x04(..)|\x66\x81\x78\x04(..)|\x66\x81\x7d\x04(..)|\x80\x7b\x06(.)|\x80\x7d\x06(.)|\x80\xf8(.)|\x80\x78\x06(.)"
+        )
```

Comment:
we'll have to find a good way to document these or maybe split them up so we can understand and extend them if needed, I think an example disassembly section per part would be great (similar to the `.data` section example above)

Reply:
I prefer to have it all together (if possible and still fairly readable)